plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -138,6 +142,8 @@
     @Inject
     private NetworkModel _netMgr;
     @Inject
+    private HostDao _hostDao;
```

Comment:
@rodrigo93 This fix was part of the jira CLOUDSTACK-9175. Maintained the same conventions as in this class. The underscore "_" is prepended to all the member variables of the class.